packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -120,6 +120,8 @@ export const MafsGraph = React.forwardRef<
                     width,
                     height,
                     position: "relative",
+                    padding: "25px 25px 0 0",
+                    boxSizing: "content-box",
```

Comment:
It's a bit unfortunate that we have to use `content-box` here, but since we're trying to preserve legacy behavior, it's probably the most straightforward and least error-prone approach.